test/test-applications/integrations/TestApplication.NServiceBus/Properties/launchSettings.json

```diff
@@ -15,14 +15,12 @@
         "OTEL_TRACES_EXPORTER": "zipkin",
         "OTEL_EXPORTER_ZIPKIN_ENDPOINT": "http://127.0.0.1:9411/api/v2/spans",
         "OTEL_LOG_LEVEL": "debug",
-        "OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED": "true",
         "OTEL_DOTNET_AUTO_HOME": "$(SolutionDir)bin\\tracer-home\\",
         "OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES": "TestApplication.*",
         "OTEL_LOGS_EXPORTER": "none",
         "OTEL_LOG_LEVEL": "debug",
         "OTEL_METRICS_EXPORTER": "otlp",
         "OTEL_METRIC_EXPORT_INTERVAL": "100",
-        "OTEL_TRACES_EXPORTER": "none"
       }
```

Comment:
`"OTEL_TRACES_EXPORTER": "console"`?
Consider similar changes for other  json files.

Reply:
fixed in 984a940f0808cf6b22c83ef6258836f9eed2549a